src/Events/Controllers/CollectController.cs

```diff
@@ -70,7 +76,8 @@ public async Task<IActionResult> Post([FromBody] IEnumerable<EventModel> model)
                     else
                     {
                         cipher = await _cipherRepository.GetByIdAsync(eventModel.CipherId.Value,
-                           _currentContext.UserId.Value);
+                           _currentContext.UserId.Value,
+                           _currentContext);
```

Comment:
(grabbed this one at random) ⚠️ Sending the entire context into repositories seems like an overreach to me.